### DISCUSSION TITLE
Transmit power setting

### DISCUSSION THREAD

**Discussion by salopette** [Koenkk/zigbee2mqtt#6283] [General]
Hello, I see the Transmit power option in the frontend.
What can one day there? What values?
I have the CC26X2R1 Ti board.
Does it bring something to increase the range?

**Comment by Koenkk** [comment]
For CC2652 the max is 5 which is the default when using https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

**Reply by timoline** [reply]
What are the possible values?
or is that hard to say, does it depends on adapter?

**Reply by Koenkk** [reply]
It depends on the adapters, it's only supported for zstack adapters in the first place. CC2652 = 5dbm, CC1352 max is = 20dbm (5dbm default)

**Reply by timoline** [reply]
ok thx, I'll add that to the schema (description)

**Reply by thiscantbeserious** [reply]
@timoline @Koenkk well the CC2652P does have an ~~20~~ 19.5 dBm amplifier built in as well.

https://www.ti.com/lit/ds/symlink/cc2652p.pdf

**Reply by Koenkk** [reply]
@thiscantbeserious true! But these adapters are not officially supported (and not sold AFAIK)

**Comment by salopette** [comment]
Thanks for the answer! Do I have to enter 5 or 5dbm?

**Reply by Koenkk** [reply]
Nothing (since 5 is the default)
